app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb

```diff
@@ -0,0 +1,279 @@
+module Hyrax
+  module Dashboard
+    module CollectionsControllerBehavior
```

Comment:
I don't think we should because you've created a whole new module here. It's more challenging to track history if we have to do code moves between files multiple times.